pkgs/by-name/ni/nixos-rebuild-ng/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  installShellFiles,
+  nix,
+  nixos-rebuild,
+  python3Packages,
+}:
+let
+  fallback = import ./../../../../nixos/modules/installer/tools/nix-fallback-paths.nix;
+  fs = lib.fileset;
+in
+python3Packages.buildPythonApplication {
+  pname = "nixos-rebuild-ng";
+  version = "0.1";
+  src = fs.toSource {
+    root = ./.;
+    fileset = fs.unions [
+      ./nixos_rebuild
+      ./pyproject.toml
+      ./tests
+    ];
+  };
```

Comment:
@emilazy ahead-of-time warning about a potential new installer test issue.

@thiagokokada see https://github.com/NixOS/nixpkgs/pull/352491 for context. It may be best to avoid using fileset for now.

Reply:
Yeah, this can’t use filesets. Please make a directory for the source.

Reply:
This is actually useless since I moved to `pkgs/by-name` anyway, thanks for the reminder.

Reply:
We do avoid `src = ./.;` in general because it means even e.g. Nix formatting changes rebuild packages. A separate source directory would be best here.

Reply:
Move all the source to `src`.